src/dotnet/dotnet.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why are these blank lines being removed? The xml here is quite dense and it is difficult to see where one logical section ends and the next begins?